tests/conftest.py

```diff
@@ -193,10 +193,21 @@ def pytest_exception_interact(node, call, report):
         if device is None:
             logger.info("Could not find device in test environment, no printing logs")
         else:
+
+            def print_device_output(instance, output):
+                """Log command output of either a MenderDevice or a MenderDeviceGroup."""
+                if isinstance(instance, MenderDevice):
+                    logger.info(output)
+                else:
+                    for dev, log in output.items():
+                        logger.info("Device %s", dev)
```

Comment:
Suggestion: This is the kind of thing one will often search for in the log. Maybe it should be a bit more unique? Maybe `Printing log of device %s`?

Reply:
Good suggestion, thanks